src/components/BookForm.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
```

Comment:
- this component is a "container" component and should be in the `src/containers/` folder